IFC4x3/PropertyEnumerations/c/PEnum_ComplementaryWorks.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_ComplementaryWorks" Name="PEnum_ComplementaryWorks" UniqueId="ecd997a9-4827-4743-b223-8c7db9deab52">
+	<Documentation>Enumeration of the complementary function possible for the drain channel.</Documentation>
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="OTHER_0SAtxxJj15wQA15w2MsIV" />
```

Comment:
Is the order important?

Reply:
No, I changed it to have a more regular Order. 48eb2f882610fb71dfd3c751fdc9e03736d59a7b